packages/cli/src/commands/pg/settings/log-lock-waits.ts

```diff
@@ -12,8 +12,8 @@ export default class LogLockWaits extends PGSettingsCommand {
   `)
 
   static args = {
-    database: Args.string(),
-    value: Args.string(),
+    database: Args.string({description: 'The config var exposed to the owning app containing the database configuration.'}),
+    value: Args.string({description: 'Boolean value indicating whether a log message should be produced when a session waits longer than the deadlock_timeout to acquire a lock.'}),
```

Comment:
Change to "boolean indicating if a message gets logged when a session waits longer than the deadlock_timeout to acquire a lock"